back/app/policies/basket_policy.rb

```diff
@@ -29,8 +29,7 @@ def destroy?
   private
 
   def check_voting_allowed(basket, user)
-    pcs = Permissions::PhasePermissionsService.new
```

Comment:
Tidied up a number of these left over references to `pcs` which was from when it was called `ParticipationContextService`